e2e/compose-env.experimental.yaml

```diff
@@ -0,0 +1,6 @@
+version: '2.1'
+
+services:
+  engine:
+      command: ['--insecure-registry=registry:5000', '--experimental']
```

Comment:
looks like my previous comment got collapsed; https://github.com/docker/cli/pull/990#discussion_r189855044

(nit); was wondering if we should prefer to use double-quotes here to be consistent with the Dockerfile format

Reply:
Ah, I don't really care so I can switch to `"` :wink: